hardhat.config.ts

```diff
@@ -43,6 +43,7 @@ const config: HardhatUserConfig = {
       "contracts/HubPool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
       "contracts/Boba_SpokePool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
       "contracts/Optimism_SpokePool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
+      "contracts/Base_SpokePool.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
```

Comment:
Added a few bits of config to the hh config in order to support base and its contracts.